src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js

```diff
@@ -311,7 +311,7 @@ export const DiseaseAnnotationsComponent = () => {
         return (
             <>
                 <DiseaseEditor 
-                    autocompleteFields={["curie", "name"]} 
+                    autocompleteFields={["curie", "name", "crossReferences.curie", "secondaryIdentifiers", "synonyms"]} 
```

Comment:
I'm not sure that synonyms is correct here @cmpich?

Reply:
Sorry never mind... these are synonyms on a DOTerm not a DA.

Reply:
The ontology loader stores synonyms as a list of strings, not synonym objects - I assumed this was intentional for non-genomic entities?